Microsoft.Azure.CosmosRepository/src/Microsoft.Azure.CosmosRepository.csproj

```diff
@@ -3,7 +3,7 @@
   <PropertyGroup>
     <Company>Microsoft Corporation</Company>
     <Product>IEvangelist Azure Cosmos Repository</Product>
-    <TargetFramework>netstandard2.0</TargetFramework>
+    <TargetFramework>netstandard2.1</TargetFramework>
```

Comment:
Unfortunately, I'd prefer to keep this on .NET Standard 2.0 because it is a much broader spectrum of users. .NET Standard 2.1 is limited with who can consume this. While I fully understand and appreciate the effort, and I too am sad about missing out on some of the features - such as async streams and nullable reference types, I'd like to keep this on 2.0 for now.

Reply:
Ok, fine by me. I'm going to keep my 2.1 version going, with formatting changes to closer match yours. I'll do another PR with just the 2.0 and significant stuff.